SU-SWS/decanter-react
# ADAPT-2296: PurgeCSS Safe Code

# READY FOR REVIEW

# Summary
- Refactors components to eliminate dynamic classes

# Needed By (Date)
- End of sprint

# Urgency
- 5/10

# Steps to Test

1. Review the code 
2. Navigate to the preview build and test out the options to make sure they all still work (Flex, Grid, Heading)
3. Pull this branch into saa_alumni and build using the JIT compiler (untested right now)

# Affected Projects or Products
- OOD Online Giving
- SAA Alumni

# Associated Issues and/or People
- ADAPT-2296
- @jdwjdwjdw FYI. 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
